src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,52 @@
 package core.basesyntax.impl;
 
+import core.basesyntax.KeyValue;
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private final Integer storageSize = 10;
```

Comment:
```suggestion
    private final int storageSize = 10;
```
we can use primitive here